charts/synthetics-job-manager/charts/node-browser-runtime/values.yaml

```diff
@@ -28,7 +28,13 @@ image:
 ## The AppArmor profile name that will be applied to the pods. If set, then the AppArmor profile must exist on the Kubernetes node(s) for this to work.
 # appArmorProfileName: ""
 
-resources: {}
+resources:
+  requests:
+    cpu: "500m"
```

Comment:
should we have a lower default cpu size? 500m is half a cpu, which can really limit how many runner pods can get scheduled to a node & result in lots of pending pods (depending on how the user sets parallelism).